pages/lessons/[name].js

```diff
@@ -1,8 +1,11 @@
 import styled from "styled-components";
-import { getLessonByName, getAllLessons } from "../../lib/db";
 import Image from "next/image";
+import dbConnect from "../../lib/dbConnect";
+import { getAllLessons } from "../../services/lessonServices";
+import { getLessonByName } from "../../services/lessonServices";
```

Comment:
i think you don't need a double import. You could write
`import { getAllLessons, getLessonByName } from "../../services/lessonServices";`